spec/spec_helper.rb

```diff
@@ -1,8 +1,25 @@
+require "codeclimate-test-reporter"
+require "simplecov"
+
+CodeClimate::TestReporter.start
+
+SimpleCov.start do
+  SimpleCov.maximum_coverage_drop 0.2
+
+  SimpleCov.start do
+    add_group  "Resources", "lib/myfinance/resources"
+    add_group  "Entities",  "lib/myfinance/entities"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.